flink-runtime/src/main/java/org/apache/flink/runtime/source/coordinator/SourceCoordinator.java

```diff
@@ -161,13 +161,24 @@ public void handleEventFromOperator(int subtask, OperatorEvent event) {
                             operatorName,
                             event);
                     if (event instanceof RequestSplitEvent) {
+                        LOG.info(
+                                "Source {} received split request from parallel task {}",
+                                operatorName,
+                                subtask);
```

Comment:
I am wondering whether this information is really helpful for a user of Flink. My spontaneous gut feeling is that it could also be logged on debug log level. This, however, already happens a couple of lines above. 

Hence, I guess the first question is whether this is information we want to expose to every user. If yes, then we should probably change the `LOG.debug` line above to not log duplicate information on debug log level.

Reply:
Let me try and deduplicate these lines.

I think in general, this is good information. Like in batch mode, when a source processes multiple splits, you don't see any progress being logged on the JM side.